src/utils/mod.rs

```diff
@@ -865,7 +865,7 @@ pub struct Signature {
 #[derive(Debug, PartialEq)]
 pub enum SignatureData {
     RsaSignature(Vec<u8>),
-    EccSignature { r: Vec<u8>, s: Vec<u8> },
+    EcdsaSignature { r: Vec<u8>, s: Vec<u8> },
```

Comment:
Should we increase the minor version number instead of the revision one because of that public change?

Reply:
Assuming we follow the same rules than if we were before `1.0.0`:

> Before you reach 1.0.0, anything goes, but if you make breaking changes, increment the minor version. In Rust, breaking changes include adding fields to structs or variants to enums.

from [here](https://doc.rust-lang.org/cargo/reference/manifest.html#the-version-field)

So I think it is fine like this and happy to see this merged!